components/agenda/src/Agenda.svelte

```diff
@@ -1082,6 +1074,16 @@
                     <p>
                       {@html DOMPurify.sanitize(event.description ?? "")}
                     </p>
+                    {#if _this.display_metadata && event.metadata}
```

Comment:
This check will return true even if the metadata object is empty, so I'd recommend changing it to this:
```suggestion
                    {#if _this.display_metadata && Object.keys(event.metadata).length}
```